server/models/UserModel/index.js

```diff
@@ -58,6 +66,34 @@ const UserModel = {
 
     return value
   },
+  hasAdmin: async (db) => {
+    const userCt = await db
+      .collection(collections.users)
+      .countDocuments({ role: "admin" })
+    return userCt !== 0
+  },
+  createFirstAdmin: async (db, dataDb) => {
```

Comment:
where is the dataDb used?

Reply:
I moved it to `update`, I'll fix. Thanks!